checkstyle/suppressions.xml

```diff
@@ -56,6 +56,11 @@
     <suppress checks="MethodCount" files="com/hazelcast/cache/impl/CacheStatisticsImpl"/>
     <suppress checks="ClassDataAbstractionCoupling" files="com/hazelcast/cache/impl/DefaultOperationProvider"/>
 
+    <!-- Cardinality -->
+    <suppress checks="JavadocMethod" files="com/hazelcast/cardinality/"/>
```

Comment:
Those package wide Javadoc suppressions are fine, but we shouldn't have them for public packages. In public packages we should solve them. All implementation details, where we are allowed to omit some JavaDoc should be in packages which contain `internal` or `impl`. For those we already have a project wide matching suppression.

So actually this one should not be needed and get fixed.